docs/architecture/cloud-native/toc.yml

```diff
@@ -43,7 +43,7 @@
       href: grpc.md
     - name: Service Mesh communication infrastructure
       href: service-mesh-communication-infrastructure.md
-  - name: Distributed Data
+  - name: Cloud-native data
```

Comment:
Can you drop this file change in this PR? Let's address this in detail introducing Cloud-Native Data and renaming the file "database-per-microservice" to "cloud-native-data.md"